### DISCUSSION TITLE
Postprocessor for Local Values

### DISCUSSION THREAD

**Discussion by rl3fz** [idaholab/moose#20142] [Q&A Tools] [answered]
Hello everyone!

Is there a way to record and display the value of a variable or material property at a certain location on the domain? It seems that I can record the mean value across the domain through the ElementAverageValue and ElementAverageMaterial Property postprocessors, or I can record the sum through the ElementIntegralVariablePostprocessor and ElementIntegralMaterialProperty postprocessors, but I'm trying to track precise values at specific points. Thank you for all that you do!

**Comment by ykvishal** [comment, marked as answer]
You can check [VectorPostprocessors System.
](https://mooseframework.inl.gov/syntax/VectorPostprocessors/index.html)  [PointValueSampler ](https://mooseframework.inl.gov/source/vectorpostprocessors/PointValueSampler.html)may be useful

**Reply by rl3fz** [reply]
This is exactly what I need, thank you!
